### DISCUSSION TITLE
Question about Example multibody.py

### DISCUSSION THREAD

**Discussion by francesco1788** [capytaine/capytaine#101] [Q&A] [answered]
Good morning,

I am running the code from the example multibody.py.
However when it comes to print(data), I only only see the initial characters of each line (see attached file).
Could anybody let me know how I could fix this issue and read the whole text, please?

![Capture](https://user-images.githubusercontent.com/91535242/135069907-1a82f743-a2d1-44bc-8842-4b1d887b40bd.PNG)

Thanks

**Comment by mancellin** [comment, marked as answer]
That is the normal behavior to only print a summary of the data contained in the dataset.

You can extract values from the dataset in several ways.
For instance, you can query a specific value with
```
data['added_mass'].sel(omega=1.0, radiating_dof="cylinder__Surge", influenced_dof="cylinder__Surge")
```
or you can extract a whole array of data (as a numpy array) with
```
data['diffraction_force'].values
```

See also [xarray's documentation](http://xarray.pydata.org/en/stable/index.html) for more possibilities of manipulation and export of this dataset.
